<launch>
    <arg name="device_id" default="060314733847"/>

    <node pkg="tf2_ros" type="static_transform_publisher" name="kinect_tf" args="0 0 2.17 0 0 -2.12 world kinect2_rgb_optical_frame"/>
    <node pkg="tf2_ros" type="static_transform_publisher" name="kinect_tf_ir" args="0 0 2.17 0 0 -2.12 world kinect2_ir_optical_frame"/>
    <node pkg="tf2_ros" type="static_transform_publisher" name="ws_center_publisher" args="-0.068, 2.554, -0.002 0.0 0.0 0.0 1 world ws_center"/>

    <include file="$(find crazyflie_kinect_localization)/launch/kinect2_bridge.launch"/>

    <include file="$(find crazyflie_kinect_localization)/launch/pc_filter.launch"/>

    <node pkg="crazyflie_kinect_localization" type="point2tf.py" name="cf1_tf" output="screen">
        <param name="cf" value="cf1"/>
    </node>

    <node pkg="crazyflie_kinect_localization" type="point2tf.py" name="cf2_tf" output="screen">
        <param name="cf" value="cf2"/>
    </node>

    <node pkg="crazyflie_kinect_localization" type="point2tf.py" name="cf3_tf" output="screen">
        <param name="cf" value="cf3"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz_view" args=" -d $(find crazyflie_kinect_localization)/rviz/default.rviz"/>
</launch>
